json5/json5
# Update to latest GitHub Pages

- Update to latest GitHub Pages
- Update index.md with latest README.md
- Remove syntax highlighting for errors since it's not reading JSON5 as JavaScript objects but instead as JavaScript blocks
- Darken font color for more contrast

You can see how this looks at [https://jordanbtucker.github.io/json5](https://jordanbtucker.github.io/json5).
